### PR TITLE
Scylla for MMS

### PR DESCRIPTION
**Pull request by rusher81572**
This is the Scylla docker files for the Mutant Monitoring System Blog

### CODE REVIEW

**Comment by rusher81572** [comment]
@tzach here is the PR for the MMS blog

**Comment by tzach** [comment]
@rusher81572 please add a readme.md to the MMS directory explaining what this directory includes.
It does not have to go into MMS details, but to explain the example:
A docker compose example for running 3 instances cluster (or similar)

**Comment by tzach** [comment]
@rusher81572 I'm geting errors:
```
 sudo docker-compose up -d
[sudo] password for tzach: 
Creating network "mms_web" with driver "bridge"
ERROR: cannot create network 66ef608ef087624595fa7587d21df492d0123c315cb0e232b3c3107717f99b18 (br-66ef608ef087): conflicts with network 2df0ff7fb9613e2d9d2ac0fd1358cf509ab23cdc3ae09ec1c8cdd59d697a6e86 (br-2df0ff7fb961): networks have overlapping IPv4
```

**Comment by rusher81572** [comment]
@tzach Added README.MD
You are getting that error from a previous Elastic Search demo. You will need to delete the network. I think it was called web or something like that 

```
docker network rm NAME
```

**Comment by tzach** [comment]
> docker network rm NAME

its not a name conflict, the problem is with the IP address (10.10.10.x) that does not work on my laptop

**Comment by rusher81572** [comment]
@tzach I fixed compose to not use static networking and switchted to DNS names instead of static IP's

**Comment by tzach** [comment]
@rusher81572 why aren't you using [Scylla docker](https://hub.docker.com/r/scylladb/scylla/)?

**Comment by rusher81572** [comment]
This way is easier for multi-dc which will be  covered in day 2 or day 3 of MMS

File: mms/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Scylla in Docker for the Mutant Monitoring Blog Series
+
+### Instructions on setting up a Scylla Cluster from this repo
+
+```
+cd mms
+docker-compose up -d
+docker exec -it docker exec -it mms_scylla-node1_1 sh
```
Comment (tzach) [inline mms/README.md]: look like a mistake

**Comment by rusher81572** [comment]
Fixed Typo in README

File: mms/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Scylla in Docker for the Mutant Monitoring Blog Series
+
+### Instructions on setting up a Scylla Cluster from this repo
+
+```
+cd mms
+docker-compose up -d
+docker exec -it mms_scylla-node1_1 sh
```
Comment (tzach) [inline mms/README.md]: cqlsh would be better

**Pull request merged**